doc/configuration.md

```diff
@@ -4,6 +4,15 @@ The Ignition configuration is a JSON document conforming to the following specif
 
 * **ignition** (object): metadata about the configuration itself.
   * **version** (string): the semantic version number of the spec. Must be at least `2.0.0`.
+  * **_config_** (objects): options related to the configuration.
+    * **_append_** (list of objects): a list of the configs to be appended to the current config.
```

Comment:
Rather fresh eyes looking at this, but do we anticipate other external configs besides `append` and `replace`? If so, maybe the scheme is a list of configs which embed the action (append, replace), with the last replace taking priority.

Prepend would be equivalent to append in Ignition terms?